apps/src/p5lab/spritelab/coreLibrary.js

```diff
@@ -1,518 +1,563 @@
-var spriteId = 0;
```

Comment:
nit: we'll want to rename this file to `CoreLibrary.js` to match since it's a class now